apps/console/src/app/pages/projects/getting-started-wizard/SelectionItem.tsx

```diff
@@ -0,0 +1,28 @@
+import { Card, Image, Space, Typography } from "antd";
```

Comment:
Let's move all the components to be under /components/wizard or something simillar